File: CPP/Program/Gaming.cpp
#ifndef GAMING_CPP
#define GAMING_CPP

#include <iostream>
#include <string>
#include "Software.cpp"
using namespace std;

class Gaming : public Software {
private:
    int fps;
    string genre;
    
public:
    Gaming() {
    }
    
    Gaming(int fps, string genre, string nama, string versi, string developer) : Software(nama, versi, developer) {
        this->fps = fps;
        this->genre = genre;
    }
    
    void setFps(int fps) {
        this->fps = fps;
    }
    
    void setGenre(string genre) {
        this->genre = genre;
    }
    
    int getFps() {
        return this->fps;
    }
    
    string getGenre() {
        return this->genre;
    }
    
    ~Gaming() {
    }
};

#endif // GAMING_CPP

File: Python/Program/Gaming.py
from Software import Software

class Gaming(Software):
    def __init__(self, fps=0, genre="", nama="", versi="", developer=""):
        super().__init__(nama, versi, developer)
        self.__fps = fps
        self.__genre = genre
    
    def set_fps(self, fps):
        self.__fps = fps
    
    def set_genre(self, genre):
        self.__genre = genre
    
    def get_fps(self):
        return self.__fps
    
    def get_genre(self):
        return self.__genre